### ISSUE TITLE
Auto-grow isn't working

### ISSUE THREAD

**Issue by xshapira** [thang-nm/Ludwigx#2]
<img width="1440" alt="Screen Shot 2021-03-25 at 15 04 31" src="https://user-images.githubusercontent.com/48856190/112539410-f756f900-8d7e-11eb-8068-5d12878c711a.png">

Hey @thang-nm, thank you for the outstanding app!

It worked great for one day and then auto-grow stopped working.

I tried to exit and run it again—didn't help. I'd appreciate if you could check it out and fix it.

Thanks,
Max

**Comment by thang-nm** [comment]
Thank Max for supporting me, I will investigate asap.

**Comment by thang-nm** [comment]
Hey @mxshapira, I've just fixed it. Please download [the latest version](https://github.com/thang-nm/Ludwigx/releases/latest).

**Comment by xshapira** [comment]
@thang-nm You're the best! Thanks a ton.

Regardless, how can I download **XApps activation tool**?

**Comment by xshapira** [comment]
@thang-nm After the update, the results have been limited to 7 max.

<img width="1440" alt="Screen Shot 2021-03-26 at 08 30 59" src="https://user-images.githubusercontent.com/48856190/112638799-9f1b0800-8e0d-11eb-81ac-79676a8e439e.png">

<img width="1440" alt="Screen Shot 2021-03-26 at 08 33 07" src="https://user-images.githubusercontent.com/48856190/112639144-eef9cf00-8e0d-11eb-8d32-1f8b849731cc.png">

**Comment by thang-nm** [comment]
Yeah @mxshapira! Just realize that Ludwig fixed their api. Now it's limited to 7 results.
(And I sent an email to you.)
